docs/react-18-and-react-native.md

```diff
@@ -0,0 +1,119 @@
+---
+id: react-18-and-react-native
+title: React 18 & React Native
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+This page describes how to use React 18 with React Native using the New React Native Architecture.
+
+> **tl;dr:** The first version of React Native compatible with React 18 is **0.69.0**. In order to use the new features in React 18 including automatic batching, `startTransition`, and `useDeferredValue`, you must migrate your React Native app to the New Architecture.
+
+## React 18 and the React Native New Architecture
+
+React 18 introduces [several new features](https://reactjs.org/blog/2022/03/29/react-v18.html) including:
+
+- Automatic batching
+- New Strict Mode behaviors
+- New hooks (`useId`, `useSyncExternalStore`)
+
+It also includes new concurrent features:
+
+- `startTransition`
+- `useTransition`
+- `useDeferredValue`
+- Full Suspense support
+
```

Comment:
`Full Suspense support`
Suspense for **Code Splitting** is actually no supported. We should mention this